sdk/src/main/java/com/hedera/hashgraph/sdk/FeeSchedule.java

```diff
@@ -0,0 +1,60 @@
+package com.hedera.hashgraph.sdk;
+
+import com.google.protobuf.InvalidProtocolBufferException;
+import java.util.List;
+import java.util.ArrayList;
+
+import org.threeten.bp.Instant;
+
+public class FeeSchedule {
+    private List<TransactionFeeSchedule> transactionFeeSchedules;
+    private Instant expiryTime;
+
+    public FeeSchedule() {
+        transactionFeeSchedules = new ArrayList<TransactionFeeSchedule>();
+    }
+
+    List<TransactionFeeSchedule> getTransactionFeeSchedules() {
+        return transactionFeeSchedules;
+    }
+    Instant getExpiryTime() {
+        return expiryTime;
+    }
+
+    FeeSchedule setExpiryTime(Instant expiryTime) {
+        this.expiryTime = expiryTime;
+        return this;
+    }
+
+
+    static FeeSchedule fromProtobuf(com.hedera.hashgraph.sdk.proto.FeeSchedule feeSchedule) {
+        FeeSchedule returnFeeSchedule = new FeeSchedule();
+        if(feeSchedule.hasExpiryTime()) {
```

Comment:
I'd recommend to do the same thing I mentioned to do with `toProtobuf()` above which is to use ternary if statements instead.